linux/system/cpu_frequency.sls

```diff
@@ -0,0 +1,10 @@
+{%- from "linux/map.jinja" import system with context %}
+{%- if system.cpu_frequency is defined %}
+
+ondemand_service_disable:
```

Comment:
Lets test it, but Filip we need to disable it on compute nodes, because it degrades power on production.